src/routes/Routes.tsx

```diff
@@ -187,7 +210,10 @@ const Routes = () => {
             <Route path={COLONY_ADVANCED_ROUTE} element={<AdvancedPage />} />
           </Route>
 
-          <Route path={ACTIONS_PAGE_ROUTE} element={<ActionDetailsPage />} />
+          <Route
+            path={OLD_ACTIONS_PAGE_ROUTE}
```

Comment:
keeping this here just in case anyone wants it as we transition uis

Reply:
I have a similar thing added here: https://github.com/JoinColony/colonyCDapp/blob/master/src/routes/Routes.tsx#L184

Note the separate `ColonyHomeOLD` component.

Please take that into account when rebasing this on master. Either combine them, or just remove one of them.